### PR TITLE
Add apiservers.config.openshift.io admission smoke test

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: test/extended/apiserver/config_admission_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:APIServer]", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLI("apiserver", exutil.KubeConfigPath())
+
+	g.It("browsers should get a redirect from /", func() {
```
Comment (stlaz) [inline test/extended/apiserver/config_admission_test.go]: copy-pasted :slightly_smiling_face: 
A better description, please

Comment (sttts) [reply test/extended/apiserver/config_admission_test.go]: done

**Comment by stlaz** [comment]
/lgtm
Should fail if the validation is indeed not happening

**Comment by sttts** [comment]
Unrelated errors.

/retest

**Comment by sttts** [comment]
docker registry error

/retest

**Pull request merged**